18F/identity-idp
# LG-10083: Update Spanish Translation for SSN View

## 🎫 Ticket
[LG-10083](https://cm-jira.usa.gov/browse/LG-10083) Implementation: Update Spanish translation for Enter your social security number page

## 🛠 Summary of changes

- Update `doc_auth.headings.ssn` and `doc_auth.info.ssn` translations for Spanish 

## 📜 Testing Plan
The text updates effect 3 views. Look at the URL to identify what view you are on. These views include... 

- [ ] the FSM SSN view -- url: /es/verify/in_person/ssn
- [ ] the non-FSM SSN view -- url: /es/verify/in_person_proofing/ssn
- [ ] doc auth SSN view -- url: /es/verify/ssn

For each view, inspect the text for the following change:

- Update this text [[h1]]] "Ingresa tu número de Seguro Social" to this new text [[[h1]]] "Ingrese su número de Seguro Social"
- Update this text [[[body]]] "Necesitamos tu número de Seguro Social para validar tu nombre, fecha de nacimiento y dirección." to this new text [[[body]]] "Necesitamos su número de Seguro Social para validar su nombre, fecha de nacimiento y dirección."

## 👀 Screenshots
<details>
<summary>Team Joy's FSM SSN View</summary>
url: /es/verify/in_person/ssn

**Before**
<img width="1124" alt="Screenshot 2023-08-07 at 9 48 44 AM" src="https://github.com/18F/identity-idp/assets/125507397/45f8181a-6b33-41c2-89f2-6fc6be648b8e">

**After**
<img width="1187" alt="Screenshot 2023-08-07 at 9 14 38 AM" src="https://github.com/18F/identity-idp/assets/125507397/99fc7e1a-e5a2-4964-959f-97a850c2bec9">
</details>

<details>
<summary>Team Joy's non-FSM SSN View</summary>
url: /es/verify/in_person_proofing/ssn

**Before**
<img width="1370" alt="Screenshot 2023-08-07 at 8 51 20 AM" src="https://github.com/18F/identity-idp/assets/125507397/85beac5b-a45d-4b54-9a85-20b02ffda3f5">

**After**
<img width="1280" alt="Screenshot 2023-08-07 at 9 18 16 AM" src="https://github.com/18F/identity-idp/assets/125507397/3a776778-30b8-49f8-a631-64817c5fd0e6">
</details>

<details>
<summary>Team Ada's SSN View</summary>
url: /es/verify/ssn

**Before**
<img width="1368" alt="Screenshot 2023-08-07 at 8 54 09 AM" src="https://github.com/18F/identity-idp/assets/125507397/4f2bbbe3-c4c1-4563-93b2-680947c53ed2">

**After**
<img width="1304" alt="Screenshot 2023-08-07 at 9 12 54 AM" src="https://github.com/18F/identity-idp/assets/125507397/3d366c50-39cc-4b6c-a9c4-17e15ece970d">
</details>
